Analysis/Tasks/lambdakzeroproducer.cxx

```diff
@@ -0,0 +1,121 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+#include "Analysis/SecondaryVertexHF.h"
+#include "DetectorsVertexing/DCAFitterN.h"
+#include "ReconstructionDataFormats/Track.h"
+#include "Analysis/RecoDecay.h"
+#include "Analysis/trackUtilities.h"
+#include "Analysis/StrangenessTables.h"
+
+#include <TFile.h>
+#include <TH2F.h>
+#include <TProfile.h>
+#include <TLorentzVector.h>
+#include <Math/Vector4D.h>
+#include <TPDGCode.h>
+#include <TDatabasePDG.h>
+#include <cmath>
+#include <array>
+#include <cstdlib>
+#include "PID/PIDResponse.h"
+#include "Framework/ASoAHelpers.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using std::array;
+
+/// Cascade builder task: rebuilds cascades
+struct lambdakzeroproducer {
+  Produces<aod::V0Data> v0data;
```

Comment:
lambdakzerofinder.cxx produces the same table, this will not work if these two workflows are intended to run together.

Reply:
This, incidentally, was completely intentional! I have now almost finished the finder using a new table with a new index + the v0data table. It is a separate use case that uses the same table... A bit tricky, but it covers something that Ruben is also interested in having.